{"date":"Wed Apr 15 2020 19:23:32 GMT+0500 (Pakistan Standard Time)","process":{"pid":5988,"uid":null,"gid":null,"cwd":"C:\\Users\\amir\\myvidly","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v10.16.3","argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\amir\\myvidly\\index.js"],"memoryUsage":{"rss":57257984,"heapTotal":41185280,"heapUsed":21900680,"external":17977470}},"os":{"loadavg":[0,0,0],"uptime":92742},"trace":[{"column":11,"file":"C:\\Users\\amir\\myvidly\\node_modules\\mongodb-core\\lib\\topologies\\server.js","function":null,"line":503,"method":null,"native":false},{"column":13,"file":"events.js","function":"Pool.emit","line":198,"method":"emit","native":false},{"column":12,"file":"C:\\Users\\amir\\myvidly\\node_modules\\mongodb-core\\lib\\connection\\pool.js","function":null,"line":326,"method":null,"native":false},{"column":20,"file":"events.js","function":"Object.onceWrapper","line":286,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Connection.emit","line":198,"method":"emit","native":false},{"column":50,"file":"C:\\Users\\amir\\myvidly\\node_modules\\mongodb-core\\lib\\connection\\connection.js","function":null,"line":245,"method":null,"native":false},{"column":20,"file":"events.js","function":"Object.onceWrapper","line":286,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Socket.emit","line":198,"method":"emit","native":false},{"column":8,"file":"internal/streams/destroy.js","function":"emitErrorNT","line":91,"method":null,"native":false},{"column":3,"file":"internal/streams/destroy.js","function":"emitErrorAndCloseNT","line":59,"method":null,"native":false},{"column":19,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":63,"method":"_tickCallback","native":false}],"stack":["MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","    at Pool.<anonymous> (C:\\Users\\amir\\myvidly\\node_modules\\mongodb-core\\lib\\topologies\\server.js:503:11)","    at Pool.emit (events.js:198:13)","    at Connection.<anonymous> (C:\\Users\\amir\\myvidly\\node_modules\\mongodb-core\\lib\\connection\\pool.js:326:12)","    at Object.onceWrapper (events.js:286:20)","    at Connection.emit (events.js:198:13)","    at Socket.<anonymous> (C:\\Users\\amir\\myvidly\\node_modules\\mongodb-core\\lib\\connection\\connection.js:245:50)","    at Object.onceWrapper (events.js:286:20)","    at Socket.emit (events.js:198:13)","    at emitErrorNT (internal/streams/destroy.js:91:8)","    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)","    at process._tickCallback (internal/process/next_tick.js:63:19)"],"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","timestamp":"2020-04-15T14:23:32.924Z"}
